libs/live-dmk-shared/src/hooks/useDeviceSessionRefresherToggle.tsx

```diff
@@ -5,19 +5,22 @@ import { activeDeviceSessionSubject } from "../config/activeDeviceSession";
 
 /**
  * Hook to disable the device session refresher when the device is connected.
- * @param {DeviceManagementKit} dmk Instance of DeviceManagementKit
+ * @param {DeviceManagementKit | null} dmk Instance of DeviceManagementKit
  * @param {boolean} ffEnabled Whether the feature flag is enabled
  * @returns a function to reenable the device session refresher or undefined when:
  *   - the feature flag is disabled
  *   - there are no active device sessions
  */
-export const useDisableDeviceSessionRefresher = (dmk: DeviceManagementKit, ffEnabled: boolean) => {
+export const useDisableDeviceSessionRefresher = (
+  dmk: DeviceManagementKit | null,
+  ffEnabled: boolean,
+) => {
   const sessionId = useRef<string>();
```

Comment:
[NOTE]: Seems like since we are in a react context, we can use `useFeature` and not pass the argument to this anymore

Reply:
Stupid comment, we can rely on the presence of DMK instead